README.md

```diff
@@ -1,24 +1,112 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-app
 
-Things you may want to cover:
+> You will create a Ruby on Rails application that allows the user to:
 
-* Ruby version
+> - register and log in, so that the data is private to them.
+> - introduce new transactions associated with a category.
+> - see the money spent on each category.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby on Rails
+- Boostrap
+- Postgres SQL
+- Rspec
+- Capybara
 
-* Database creation
+## Live Demo (if available)
```

Comment:
- [ ] As part of the project requirements, please deploy the application using Heroku, Netlify, or GitHub Pages and include a link to the deployed project in your README 👍

Reply:
* These requested change hasn't been fixed kindly fix this as its a[ requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20project%20should%20be%20deployed%20and%20accessible%20online.)